src/flamegraph/flamegraph.js

```diff
@@ -330,17 +329,20 @@ function search(term) {
     var maxwidth = 0;
     for (var i = 0; i < el.length; i++) {
         var e = el[i];
+        if (e.classList.contains("hide")) {
```

Comment:
This feels like an unrelated change?

Reply:
Could be. I was thinking of this PR as a fix for getting incorrect values from ctrl+f, especially when zoomed in. I'm not sure if this is fixing another bug altogether or if I've misunderstood what ctrl+f is supposed to do, but it's in here because I realized that storing the sample counts isn't sufficient to get the matched % I've been expecting.

Reply:
Oh, I skimmed over your original PR comment too quickly. Yes, I think this does indeed fix a bug, and happy to take it as part of this change :+1: